Rover/mode.cpp

```diff
@@ -353,8 +353,12 @@ bool Mode::stop_vehicle()
     // send to motor
     g2.motors.set_throttle(throttle_out);
 
-    // do not attempt to steer
-    g2.motors.set_steering(0.0f);
+    // do not turn while slowing down
+    float steering_out = 0.0;
+    if (!stopped) {
+        steering_out = attitude_control.get_steering_out_rate(0.0, g2.motors.limit.steer_left, g2.motors.limit.steer_right, rover.G_Dt);
+    }
+    g2.motors.set_steering(steering_out * 4500.0);
```

Comment:
nice to use SERVO_MAX here (not a blocker for PR)

Reply:
Txs for the review.  It seems in Rover (from a peek here and there) we always just use 4500.0 instead of the define.  So for better or worse I think I'll stick with this hard coded number.